src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs

```diff
@@ -95,7 +95,16 @@ private void InitializeResetInteractiveFromProjectCommand(OleMenuCommandService
 
                     var vsInteractiveWindow = _interactiveWindowProvider.Open(instanceId: 0, focus: true);
 
-                    resetInteractive.Execute(vsInteractiveWindow, LanguageName + " Interactive");
+                    EventHandler focusWindow = null;
```

Comment:
This has been extracted from the original `ResetInteractive.Execute` command to remove dependency on `IVsInteractiveWindow`.